src/D2L.Security.OAuth2/Keys/Caching/InMemoryPublicKeyCache.cs

```diff
@@ -37,12 +37,12 @@ MemoryCache cache
 			);
 		}
 
-		D2LSecurityToken IInMemoryPublicKeyCache.Get( string srcNamespace, Guid keyId ) {
+		D2LSecurityToken IInMemoryPublicKeyCache.Get( string srcNamespace, string keyId ) {
 			var result = m_cache.Get( BuildCacheKey( srcNamespace, keyId ) ) as D2LSecurityToken;

```

Comment:
I believe `MemoryCache` is case-sensitive / uses HashCode, so should be okay.

Reply:
Yup, uses Ordinal: https://referencesource.microsoft.com/#System.Runtime.Caching/System/Caching/MemoryCacheKeyEqualityComparer.cs